schema.sql

```diff
@@ -6,3 +6,7 @@ CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL(5,2)
 );
+
+ALTER TABLE vet_clinic
+ADD COLUMN species VARCHAR(30);
+SELECT * FROM vet_clinic;
```

Comment:
👍 You're making good progress on your project. However, I noticed a specific aspect related to the SQL files that needs attention in line with the project requirements, as previously mentioned in the previous milestone. 

- [ ] As a best practice, the `schema.sql` file should contain everything related to creating or changing the table structure. This file is primarily for defining the database schema. On the other hand, the `queries.sql` file should be reserved for queries that help analyze data or retrieve specific information from the database. In your case, This SQL query that retrieves all entities from the `vet_clinic` database should be placed in the `queries.sql` file instead of the `schema.sql` file. 

By following this structure carefully, you will align your code with the documentation and project requirements more effectively. Keep up the good work! 👍📜🐘